src/content/contributing/translation-program/blog-translations/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: EF blog translations
+lang: en
+description: How to contribute by translating the Ethereum Foundation blog
+sidebar: true
+---
+
+# EF blog translations {#blog-translations}
+
+The [Ethereum Foundation blog](https://blog.ethereum.org/) is the primary publication tool for a lot of EF-supported teams. Several teams regularly use the blog to publish announcements, updates, reports, roundups, and other public communication.
+
+In line with some key values from the [ethereum.org Translation Program mission and vision](/contributing/translation-program/mission-and-vision) - accessibility, inclusivity, empowerment, and building the global Ethereum community - we want to translate the Ethereum Foundation blog posts in as many languages as possible.
+
+The blog translation project is open and anyone can contribute.
+If you want to get involved and help translate blog posts into your language, read on to learn how to join the project in Crowdin and start translating.
+
+## How to translate {#how-to-translate}
+
+<InfoBanner shouldCenter emoji=":light_bulb:">
+  We encourage you to check out the ethereum.org Translation style guide. It contains some of the most important guidelines, instructions, and tips for translators to help localize the blog. While originally written for ethereum.org translators, most of the style guide is relevant to blog translations as well.
```

Comment:
- I'd personally use translate > localize because more people will understand it.

```suggestion
<InfoBanner shouldCenter emoji=":light_bulb:">

  We encourage you to check out the ethereum.org translation style guide. It contains some of the most important guidelines, instructions, and tips for translators to help translate the blog. While initially written for ethereum.org translators, most of the style guide is also relevant to blog translations.
```